snapcraft/tests/test_commands_list_plugins.py

```diff
@@ -23,25 +24,45 @@
 
 class ListPluginsCommandTestCase(tests.TestCase):
 
+    # plugin list when wrapper at MAX_CHARACTERS_WRAP
+    default_plugin_output = (
+        "ant        catkin  copy  jdk     kernel  maven  nodejs   python3  tar-content\n"  # noqa
+        "autotools  cmake   go    kbuild  make    nil    python2  scons  \n")  # noqa
```

Comment:
I'm a fan of 120 characters, but let's not open this debate here :p

Here, I used the #noqa stenza as the important part is to be able to check visually as well that we have a column output, making clear the test is about that. I have no strong opinion if you think it's still readable when being broken in multiple lines, just tell me :)